handbook/engineering/core-application/index.md

```diff
@@ -63,14 +63,14 @@ Feel free to tag `@core-app` on Slack or anyone directly as and when required. I
 ## Members
 
 <!-- Due to the markdown renderer that we use, the indentation here is sensitive. If you want to change the indentation, check that it renders correctly locally with `make serve` -->
```

Comment:
VSCode changed the indentation on save for me. I checked with `make serve` and it all looks fine